tests/installation/bootloader_s390.pm

```diff
@@ -192,17 +212,17 @@ EO_frickin_boot_parms
     else {
         $display_type = "VNC";
     }
+    if (!get_var('AGAMA_TEST')) {
```

Comment:
then we can use `unless` here, it is simple enough.

Reply:
updated.